doc/data_checklist.md

```diff
@@ -0,0 +1,68 @@
+## Working With Large SQL Data Dumps
+
+Working with large SQL dump files can be a high risk endeavor;
+the size of some files can cause a small mistake early on to 
+trash hours of work dumping, zipping, uploading and importing
+the file. To ensure we are measuring twice and cutting once,
+copy and paste this checklist into issues that involve loading
+new SQL data. For example, issues like #1531, #2532, #1408.
```

Comment:
Not sure why, but Github didn't turn these into links to the issues, so you may have to link them manually.

Reply:
Thanks, I'll make them proper links.